packages/excalidraw/element/binding.ts

```diff
@@ -142,28 +152,32 @@ const bindOrUnbindLinearElementEdge = (
 export const bindOrUnbindSelectedElements = (
   elements: NonDeleted<ExcalidrawElement>[],
 ): void => {
+  const elementsMap = arrayToMap(elements);
```

Comment:
passed in `elements` are just a subset (e.g. selected elements) — while it seems to work here (I haven't done thorough checks tho), I'm not super comfortable with this because at the other end it's going into  helpers like `getPointAtIndexGlobalCoordinates()` and then to `getElementAbsoluteCoords()` and I'm thinking those helpers (`getElementAbsoluteCoords()` especially) shouldn't be taking `ElementsMap` (subset) and instead `SceneElementsMap`...